spdk.spec

```diff
@@ -62,7 +63,7 @@ BuildRequires: python
 %endif
 
 # Install dependencies
-Requires: dpdk = 19.11.6
+Requires: dpdk = 21.05
```

Comment:
I tend to doubt this is necessary, and is just historic cruft, particularly if spdk is just using libraries from dpdk and not any kind of tools, etc.  If that's the case, it's generally better to let DNF (or YUM, nominally) figure out the dpdk library RPMs needed.

It would be worth seeing if it can be removed.